ui_framework/components/expression/expressionItemPopover.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const POPOVER_ALIGN = [
+  'left',
+  'right',
+];
+
+/*
+With some modification copy-pasted from
+https://stackoverflow.com/questions/32553158/detect-click-outside-react-component
+*/
+const outsideClickNotifier = WrappedComponent => class extends React.Component {
+  constructor(props) {
```

Comment:
I wanted to separate the stateless from the statefull.
  Stateless is KuiExpressionItemPopover, statefull is outsideClickNotifier.
  Probably let outsideClickNotifier reuse. If it's not worth to do it
  I would merge it into KuiExpressionItemPopover.